SonarQube.TeamBuild.Integration/CoverageReportConverter.cs

```diff
@@ -101,7 +101,7 @@ private static string GetExeToolPath(ILogger logger)
                 
                 if (versionToolMap.Count > 1)
                 {
-                    logger.LogMessage(Resources.CONV_DIAG_MultipleVsVersionsInstalled, string.Join(", ", versionToolMap.Keys));
+                    logger.LogInfo(Resources.CONV_DIAG_MultipleVsVersionsInstalled, string.Join(", ", versionToolMap.Keys));
```

Comment:
I'd have said debug. It's not something the user would care about in normal cases (i.e. when everything is working).